test/distributions/integer.test.ts

```diff
@@ -0,0 +1,40 @@
+import test from 'ava'
+import seedrandom from 'seedrandom'
+import random from '../../src/random'
+import inDelta from '../_in-delta'
+
+test('random.int() produces numbers', (t) => {
+  const r = random.clone(seedrandom('ZDJjM2IyNmFlNmVjNWQwMGZkMmY1Y2Nk'))
+  for (let i = 0; i < 10000; ++i) {
+    const v = r.int()
+    t.is(typeof v, 'number')
```

Comment:
same with these; let's make sure that the optional `min` and `max` are being interpreted correctly.